coffeelint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "max_line_length": {
+    "level": "ignore"
```

Comment:
Normally I would say this should be 120 at most, but I plan on re-writing this in JS anyway so this is fine.